Specs/Data/Terrain/assets_cesium_com/layer.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Cesium World Terrain data should go in `Specs/Data/CesiumTerrainTileJson` and ArcGIS data should go in `Specs/Data/ArcGIS`. I'm having trouble coming up with a naming scheme to differentiate from other files in those folders, so how about a subfolder with the tile coords as the name (e.g. `9_214_379/`)